persistence/remote/src/main/java/org/infinispan/persistence/remote/configuration/RemoteStoreConfiguration.java

```diff
@@ -52,13 +52,15 @@ public Object readAttributeValue(String enclosingElement, AttributeDefinition at
          .immutable().build();
    static final AttributeDefinition<String> REMOTE_CACHE_NAME = AttributeDefinition.builder("remoteCacheName", "").immutable().xmlName("cache").build();
 
+   static final AttributeDefinition<String> URI = AttributeDefinition.builder("uri", null,String.class).immutable().build();
```

Comment:
The formatting needs fixed in this file so that there is a space after `null,`. The same applies to `,URI` on L63.